decidim-initiatives/spec/controllers/decidim/initiatives/admin/initiatives_types_controller_spec.rb

```diff
@@ -9,8 +9,8 @@ module Admin
         routes { Decidim::Initiatives::AdminEngine.routes }
 
         let(:organization) { create(:organization) }
-        let(:admin_user) { create(:user, :confirmed, :admin, organization:) }
-        let(:user) { create(:user, :confirmed, organization:) }
+        let(:admin_user) { create(:user, :confirmed, :admin, admin_terms_accepted_at: Time.current, organization:) }
```

Comment:
The `:admin` trait has already the admin terms accepted, so we don't need to explicitly define it here.
```suggestion
        let(:admin_user) { create(:user, :confirmed, :admin, organization:) }
```